### DISCUSSION TITLE
Initial support for Android 12

### DISCUSSION THREAD

**Discussion by tiann** [android-hacker/VAExposed#1109] [General]
[0.22.0.zip](https://github.com/android-hacker/VirtualXposed/files/8849833/0.22.0.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/android-hacker/VirtualXposed/releases/tag/0.22.0'>Initial support for Android 12</a>.</em>

**Comment by zbdhdb** [comment]
半天都打不开xposed installer怎么回事

**Reply by skiy77** [reply]
同样的打不开

**Reply by zouyaoji** [reply]
+1

**Reply by Dboy233** [reply]
+1

**Reply by koalaauto** [reply]
+1

**Reply by jerryandtom007** [reply]
换了新手机后一直在等更新，还是打不开xposed installer   =。=

**Reply by ritalee11** [reply]
打不开添加进去的应用，也打不开xposed installer。。。。
Pixel 3XL Android12

**Reply by casperworld** [reply]
+1

**Reply by qindaxianren** [reply]
一直提示打开失败

**Reply by qindaxianren** [reply]
20的版本有闪退的问题

**Reply by 18739220686** [reply]
你好，闪退

**Comment by azzhu** [comment]
can not open any app in virtualxposed.

android 12, mui 13.0.5, redmi k40,

**Reply by YouJiRobbery** [reply]
+1

**Reply by BlueCh4n** [reply]
+1

**Reply by onthewayals** [reply]
same

**Reply by ErmanYY** [reply]
+1

**Reply by koalaauto** [reply]
+1

**Reply by Pedro-null** [reply]
+1(redmi k50, android 12, miui 13.0.21)

**Reply by PhillMed** [reply]
+1(realme 8, android 12, realmiUi 3.0), but it worked only for the first 2-3 days, now in no way, a black screen when trying to open xposed installer

**Reply by ugvfpdcuwfnh** [reply]
+1

**Reply by wiz64** [reply]
+1 poco x4 pro
Also unable to open Manage Modules section

**Comment by bbrainfuckk** [comment]
I'm testing VirtualXposed for GameGuardian (0.22); Xposed Installer is stuck on loading, also games are not opening (ColorOS v12.1 -  Android 12)

**Comment by YouJiRobbery** [comment]
为什么一直加载？

**Comment by nonand12** [comment]
Hello can you create for 32bit xposed? 😢

**Reply by Buchinyann** [reply]
+1

**Comment by briantongcs** [comment]
Most games such as Jurassic World and Simcity won't even start properly. It's just crashing before loading up

**Comment by itplaying** [comment]
环境：Android 9.0， API 28
1、打开xposed installer，app提示“打开应用de.robv.android.xposed.installer失败:(”
2、运行日志报错，错误日志如下
W/System.err: android.content.pm.PackageManager$NameNotFoundException: de.robv.android.xposed.installer
W/System.err:     at android.app.ApplicationPackageManager.getApplicationInfoAsUser(ApplicationPackageManager.java:431)
W/System.err:     at android.app.ApplicationPackageManager.getApplicationInfo(ApplicationPackageManager.java:411)
W/System.err:     at android.app.ApplicationPackageManager.getResourcesForApplication(ApplicationPackageManager.java:1554)
W/System.err:     at com.google.android.apps.nexuslauncher.CustomIconProvider.getRoundIcon(CustomIconProvider.java:125)
W/System.err:     at com.google.android.apps.nexuslauncher.CustomIconProvider.getIcon(CustomIconProvider.java:115)
W/System.err:     at com.android.launcher3.IconCache.getFullResIcon(IconCache.java:182)
W/System.err:     at com.android.launcher3.IconCache.getFullResIcon(IconCache.java:178)
W/System.err:     at com.android.launcher3.IconCache.cacheLocked(IconCache.java:537)
W/System.err:     at com.android.launcher3.IconCache.getTitleAndIcon(IconCache.java:479)
W/System.err:     at com.android.launcher3.IconCache.getTitleAndIcon(IconCache.java:451)
W/System.err:     at com.android.launcher3.AllAppsList.add(AllAppsList.java:81)
W/System.err:     at com.android.launcher3.model.LoaderTask.loadAllApps(LoaderTask.java:821)
W/System.err:     at com.android.launcher3.model.LoaderTask.run(LoaderTask.java:164)
W/System.err:     at android.os.Handler.handleCallback(Handler.java:907)
W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:105)
W/System.err:     at android.os.Looper.loop(Looper.java:216)
W/System.err:     at android.os.HandlerThread.run(HandlerThread.java:65)
W/HwResourcesImpl: icon in Original image 2131558400#de.robv.android.xposed.installer is too large.iconSize:234,w*h: 192*192 . so cut,sDefaultSizeWithoutEdge=232; r=Rect(13, 22 - 178, 169) ; isZoomOutIcon=false
W/System: A resource failed to call release. 
W/System: A resource failed to call release.

**Comment by sunnybob1** [comment]
redmi 7pro，OS:android miui 12.5.4，安装完打开应用，
![Screenshot_2022-07-04-12-02-58-403_io va exposed64](https://user-images.githubusercontent.com/108647341/177082595-adfdbf7f-e110-45ef-a9d0-0ad721f67fc5.jpg)
VirtualXposed0.22.0安装打开提示:打开应用de.robv.android.xposed.installer 失败

**Reply by qindaxianren** [reply]
我也是这个问题，这个软件的bug太多了，还不知道什么时候才可以修复

**Comment by sunnybob1** [comment]
麻烦看一下什么问题，看是否可以解决

**Comment by jianzhichangkong** [comment]
打不开xposed安装模块

**Comment by GuangYu-yu** [comment]
求更新，无法打开任何应用，包括模块管理器。只发现内置的文件管理和microG能打开

**Comment by GuangYu-yu** [comment]
克隆app可以添加搜索框和显示包名。

**Comment by GuangYu-yu** [comment]
求求了快更新吧，MIUI 14和Android 13都快来了

**Comment by lzus** [comment]
一样的，android 12上xposed installer一直loading 
<img width="340" alt="image" src="https://user-images.githubusercontent.com/48306281/184587073-3e22ec87-e2c1-4e02-a142-f8e37265df54.png">

**Comment by whiteday13jin** [comment]
![photo_2022-09-13_14-43-36](https://user-images.githubusercontent.com/113036420/189829476-4e264836-0547-4640-85a1-e0aee8c0e8f3.jpg)
卡在加载...似乎安卓12都是这个问题，缺权限吗？

**Reply by Juan08jjjb** [reply]
Download app

**Comment by yipingbubu** [comment]
求更新安卓12进不去

**Reply by LGF654** [reply]
我提个建议 安卓12好多应用都打不开 几乎为0 安装正常 不能正常打开 还有就是大多数软件都是32位的 64位无法用32位的软件 建议出个32位版本的 单独弄个32位插件也可以 主要是不能用32位 旧版本有32位 但是不支持安卓12

**Comment by linghua233** [comment]
用不了

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by Alphandbet** [comment]
How can I make a app crypto

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by ffmmsjluj** [comment]
>要多一个32位VirtualXposed12的系统不支持32位的VirtualXposed

**Comment by clpluie** [comment]
这个还在有人维护么

**Comment by yipingbubu** [comment]
作者大大好没更新,具体什么时候会更也不知道



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年1月4日(星期三) 下午4:38
收件人: ***@***.***&gt;; 
抄送: "Linus ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [android-hacker/VirtualXposed] Initial support for Android 12 (Discussion #1109)





 
这个还在有人维护么

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by ZH0317** [comment]
Would Harmony OS 3.0 be able to use this app?

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by winter862** [comment]
nice

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by SSL-ACTX** [comment]
Not working at all in my Infinix X669 (30i). Nothing's happening when opening the apps.

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by liuhai7435** [comment]
怎么不更新了

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by felfhfl** [comment]
0.22  android 9
arm v7 - apk
[virtualxposed-0.22.apk.zip](https://github.com/user-attachments/files/17804997/virtualxposed-0.22.apk.zip)

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by didiaodanding** [comment]
您好，您的邮件已收到，谢谢。                                    --------李峰

**Comment by byscourge** [comment]
android 13?

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.

**Comment by didiaodanding** [comment]
您好，您的邮件已收到，谢谢。                                    --------李峰

**Comment by GWARslave119** [comment]
Hi, I was curious if there are plans for supporting a/oos15 in the future?

**Comment by jingguolei** [comment]
这是来自QQ邮箱的自动回复邮件景国磊 已收到您的邮件
This is an automatic reply, confirming that your e-mail was received.Thank you.
